WordPress/Sniffs/WhiteSpace/CastStructureSpacingSniff.php

```diff
@@ -41,37 +37,25 @@ public function register() {
 	 * @return void
 	 */
 	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
-		$tokens = $phpcsFile->getTokens();
+		parent::process( $phpcsFile, $stackPtr );
 
-		$content  = $tokens[ $stackPtr ]['content'];
-		$expected = str_replace( ' ', '', $content );
-		$expected = str_replace( "\t", '', $expected );
-
-		if ( $content !== $expected ) {
-			$error = 'Cast statements must not contain whitespace; expected "%s" but found "%s"';
-			$data  = array(
-				$expected,
-				$content,
-			);
-			$phpcsFile->addWarning( $error, $stackPtr, 'ContainsWhiteSpace', $data );
```

Comment:
This is the old definition of the `ContainsWhiteSpace` check which differs from the upstream one by using `addWarning()` instead of `addError()`